From e3a95c62bda2d734c78dc55aa0943c14b6fab019 Mon Sep 17 00:00:00 2001
Message-Id: <RFC-patch-4.4-e3a95c62bda-20220711T110019Z-avarab@gmail.com>
In-Reply-To: <RFC-cover-0.4-00000000000-20220711T110019Z-avarab@gmail.com>
References: <YstJl+5BPyR5RWnR@tapette.crustytoothpaste.net>
	<RFC-cover-0.4-00000000000-20220711T110019Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 11 Jul 2022 11:51:50 +0200
Subject: [RFC PATCH 4/4] .clang-format: don't indent "goto" labels
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This change is a slightly mixed bag, we have a lot of "goto" labels
that are indented by exactly one space.

Before & after this change running "make style-all-diff-apply" will
yield:

	509 files changed, 13042 insertions(+), 12745 deletions(-)
	510 files changed, 13039 insertions(+), 12742 deletions(-)

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 .clang-format | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/.clang-format b/.clang-format
index 5a106d959be..56d7e8f9def 100644
--- a/.clang-format
+++ b/.clang-format
@@ -148,6 +148,9 @@ SpacesInSquareBrackets: false
 # clang-format 12.
 BitFieldColonSpacing: None
 
+# Do not indent "goto" labels, they should be flushed left.
+IndentGotoLabels: false
+
 # Insert a space after '{' and before '}' in struct initializers
 Cpp11BracedListStyle: false
 
-- 
2.37.0.913.g189dca38629

